lib/Ed25519VerificationKey2020.js

```diff
@@ -30,11 +38,32 @@ export class Ed25519VerificationKey2020 extends LDKeyPair {
   constructor(options = {}) {
     super(options);
     this.type = SUITE_ID;
-    this.publicKeyMultibase = options.publicKeyMultibase;
-    if(!this.publicKeyMultibase) {
+    const publicKeyMultibase = options.publicKeyMultibase;
+    const privateKeyMultibase = options.privateKeyMultibase;
+
+    if(!publicKeyMultibase) {
       throw new TypeError('The "publicKeyMultibase" property is required.');
     }
+
+    if(!publicKeyMultibase || !_isValidKeyHeader(
+      publicKeyMultibase, MULTICODEC_ED25519_PUB_HEADER)) {
+      throw new Error(
+        '`publicKeyMultibase` has invalid byte header: ' +
+        `"${publicKeyMultibase}".`);
+    }
+
+    if(privateKeyMultibase && !_isValidKeyHeader(
+      privateKeyMultibase, MULTICODEC_ED25519_PRIV_HEADER)) {
+      throw new Error(
+        '`privateKeyMultibase` has invalid byte header: ' +
+        `"${privateKeyMultibase.slice(0, 4)}".`);
```

Comment:
I don't think this should be here at all. What if someone passes a private key directly -- then 4 bytes will be leaked.

Reply:
Fixed in 551586ca4f1474bb6cbcc725d6fc7e054684287a.